tests/core/pyspec/README.md

```diff
@@ -7,27 +7,30 @@ With this executable spec,
  test-generators can easily create test-vectors for client implementations,
  and the spec itself can be verified to be consistent and coherent through sanity tests implemented with pytest.
 
-## Building
+## Dev Install
 
-To build the pyspec: `python setup.py build`
- (or `pip install .`, but beware that ignored files will still be copied over to a temporary dir, due to pip issue 2195).
-This outputs the build files to the `./build/lib/eth2spec/...` dir, and can't be used for local test running. Instead, use the dev-install as described below. 
+First, create a `venv` and install the developer dependencies (`test` and `lint` extras):
 
-## Dev Install
+```shell
+make install_test
+```
 
-All the dynamic parts of the spec are automatically built with `python setup.py pyspecdev`.
-Unlike the regular install, this outputs spec files to their original source location, instead of build output only.
+All the dynamic parts of the spec are built with:
 
-Alternatively, you can build a sub-set of the pyspec with the distutil command: 
-```bash
-python setup.py pyspec --spec-fork=phase0 --md-doc-paths="specs/phase0/beacon-chain.md specs/phase0/fork-choice.md" --out-dir=my_spec_dir
+```shell
+(venv) python setup.py pyspecdev
```

Comment:
(Not an objection of writing the full command, just double-checking)
It's as same as `make pyspec` Makefile command, right?

Reply:
Yea, `make pyspec` is the same. However, I do want to emphasize the difference between the developer-pyspec command, and the thing that runs as part of the build process.